app/lib/katello/resources/candlepin/pool.rb

```diff
@@ -0,0 +1,44 @@
+require 'katello/resources/candlepin'
+
+module Katello
+  module Resources
+    module Candlepin
+      module PoolRestClient
+        class << self
+          def find(pool_id)
+            pool_json = self.get(path(pool_id), self.default_headers).body
```

Comment:
I know this is WIP, but I'm curious if leaving the HTTP client methods `get`, etc up to the person including this module is your final design?

Reply:
Short answer: yes. It's kind of meant to be used with the `Katello::Resources::Candlepin::CandlepinResource` classes and subclasses, which all implement the `RestClient` methods.